pkg/controller/registry/reconciler/reconciler.go

```diff
@@ -2,14 +2,19 @@
 package reconciler
 
 import (
+	"fmt"
+	"hash/fnv"
 	"strings"
 
+	corev1 "k8s.io/api/core/v1"
```

Comment:
nit, not blocking: there's already an existing k8s.io/api/core/v1 import declared below this.